src/data/ACTIONS/root/NIN.ts

```diff
@@ -116,6 +116,14 @@ export const NIN = ensureActions({
 		speedAttribute: Attribute.SKILL_SPEED,
 	},
 
+	PHANTOM_KAMAITACHI_BUNSHIN: {
+		id: 25775,
+		name: 'Phantom Kamaitachi',
+		icon: 'https://xivapi.com/i/002000/002929.png',
+		onGcd: true,
```

Comment:
nit, but if this is the pet skill, it's not really onGcd or affected by a speedAttribute

Reply:
It seemed correct to me to keep the attributes here because this is kind of a weird case. We get `action` events with the player skill ID (25774) but the `damage` events come through as 25775 instead, as two halves of the same whole. It's not a thing the "pet" here does independently, it's only ever triggered manually when the player hits the Phantom Kamaitachi button, and it respects SkS/is a GCD.

Reply:
SMN pets have some similar cases (the actual summons this expansion and egi assaults last expansion).  Not something I'd recommend blocking over either way though.